src/Administration/Resources/app/administration/src/module/sw-order/page/sw-order-detail/index.js

```diff
@@ -430,24 +420,27 @@ export default {
             }
         },
 
+        /**
+         * @deprecated tag:v6.8.0 - Will be replaced by `saveAndReload`
+         */
         onSaveAndReload() {
-            Store.get('swOrderDetail').setLoading([
-                'order',
-                true,
-            ]);
+            return this.saveAndReload();
+        },
 
-            return this.orderRepository
-                .save(this.order, this.versionContext)
-                .then(() => this.reloadEntityData())
-                .catch((error) => {
-                    this.onError('error', error);
-                })
-                .finally(() => {
-                    Store.get('swOrderDetail').setLoading([
-                        'order',
-                        false,
-                    ]);
-                });
+        async saveAndReload(afterSaveFn = null) {
```

Comment:
I don't really see the point of this rename. All other methods are still called `on...`, so this just leads to an inconsistent state.

Reply:
I know, but keeping the old method means that any override that does not yet accept the parameter will break the promotion module. I think it is highly likely, as this is one of the core methods